### PR TITLE
Update Gradle, Kotlin 1.3, webextensions v0.3, README

### PR DESCRIPTION
**Pull request by jmfayard**
Basically the same pull-request as 
https://github.com/cypressious/second-firefox-extension-kotlin/pull/2

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -2,8 +2,14 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <bytecodeTargetLevel>
+      <module name="content_script_main" target="1.8" />
```
Comment (jmfayard) [inline .idea/compiler.xml:5]: Is it on purpose that parts of `.idea/` are not in gitignore?

Comment (cypressious) [reply .idea/compiler.xml:5]: Yes, some of the files in `.idea` are useful to check in. See https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore

**Comment by cypressious** [comment]
I just noticed that the change in https://github.com/cypressious/webextension-search-kotlin-docs/blob/a22d63fc839276317cc298c4c43fe29490bfc8c3/.idea/misc.xml#L70 is problematic if you don't have a JDK configured with the exact same name. Maybe we should exclude those files from git after all.

**Comment by jmfayard** [comment]
I have always excluded the whole `.idea` repo and never had a problem. 
If there is some important information inside the idea folder it should be configured by the Gradle build IMHO (ex: using the ktlint plugin for the codestyle)

**Pull request merged**